virbos-setup

```diff
@@ -214,6 +214,11 @@ getlayout() {
 			ask_nodefault "What partition would you like to use for swap space?" "$partitions"
 			partition_swap="$answer"
 		fi
+		ask_yn "Do you want to use a different partition for /home?"n
+		if [ "$answer" = 'y' ];then
+			ask_nodefault "What partition would you like to use for swap space?" "$partitions"
```

Comment:
This should say something like "What partition would you like to use in place of /home?"

Reply:
resolved